### DISCUSSION TITLE
Author Site Links - Not appearing.

### DISCUSSION THREAD

**Discussion by Richieone13** [mmistakes/minimal-mistakes#2710] [Q&A] [answered]
Hi - I'm new to using Jekyll,

My site and repo
richieone13.github.io
https://github.com/Richieone13/richieone13.github.io

I'm using this as a [template](https://github.com/academicpages/academicpages.github.io) which is a fork of https://github.com/mmistakes/minimal-mistakes. (From what I can see they removed the links) How come the links are not coming up for author site?

I have checked the _config.yml file to ensure the links: are added back in but it's still not connecting.

Reference:
https://mmistakes.github.io/minimal-mistakes/docs/authors/

Thanks,

Richie

**Comment by mmistakes** [comment, marked as answer]
They don’t show up for you because you have a ad/content blocker installed in your web browser and is removing the Font Awesome icons and links. 

Check the closed issues for more context as this has been covered before.
